tests/headless/test_endpoints.py

```diff
@@ -248,7 +248,7 @@ def test_api_doc_404(site_url):
 @pytest.mark.nondestructive
 @pytest.mark.parametrize('expected,accept,cookie,param',
                          LOCALE_SELECTORS.values(),
-                         ids=LOCALE_SELECTORS.keys())
+                         ids=list(LOCALE_SELECTORS))
```

Comment:
Sad that pytest couldn't cope with this being a generator but here we are.

Reply:
I agree!